### ISSUE TITLE
Help with Parametrization

### ISSUE THREAD

**Issue by laurenzheidrich** [PRBonn/vdbfusion#44]
Hi & thanks for your great repository!

Using Kitti Dataset I get very good results. Now I am trying to run your pipeline on a custom dataset. While the fine detail of cars is astonishing, the reconstruction of large structures such as the street is quite bad. I have tried different parametrizations but the problem stays the same.

Here are two screenshots of the resulting reconstructed mesh

<img width="1434" alt="image" src="https://github.com/PRBonn/vdbfusion/assets/76972145/f9cc7d17-5eb1-4242-8e39-5c22fecb2eb0">
<img width="728" alt="image" src="https://github.com/PRBonn/vdbfusion/assets/76972145/88775dca-ed22-4aee-bdef-7c2bd39306da">

And here is a screenshot of the the merged point cloud: 

<img width="1027" alt="image" src="https://github.com/PRBonn/vdbfusion/assets/76972145/fb2f0a25-af45-48f9-a3d0-6a9811576f15">

As you can see the merged pointcloud (consisting of 50 scans) is very dense and provides great detail, but the mesh is full of holes and missing parts. I have used the standard Kitti parametrization you provided, but played around with the voxel_size, sdf_trunc, min_weight, fill_holes etc. in the range of [x0.1, x10] and the results were always almost the same.

Can you point me in any direction of what I could change in the parametrization / setting  etc.?

**Comment by laurenzheidrich** [comment]
Translating my point clouds down by 1.5-2.5 tremendeously changed the reconstruction result to the better. I am closing this issue :)

**Comment by lijing137** [comment]
> Translating my point clouds down by 1.5-2.5m tremendeously changed the reconstruction result to the better. I am closing this issue :)

I encountered a similar issue. Could you please explain it in more detail? Is it about lowering the height of the LiDAR?

**Comment by laurenzheidrich** [comment]
Yeah so when I looked at my scans in comparison to Kitti, I saw that the ground points are around 1.5m higher than the ground points of Kitti. So as a last straw to grasp, I just translated all my scans down by 1.5m along the z-axis so that the ground points of my scans and kitti scans are somewhat aligned.

after doing that, VDB started to work perfectly. I don’t know why tbh, not sure if there is some parameter set in their code that detects the ground at a certain pre-defined height or something. Would be great if the authors could say anything about this.

**Comment by lijing137** [comment]
您好，我已收到邮件

**Comment by lijing137** [comment]
> Yeah so when I looked at my scans in comparison to Kitti, I saw that the ground points are around 1.5m higher than the ground points of Kitti. So as a last straw to grasp, I just translated all my scans down by 1.5m along the z-axis so that the ground points of my scans and kitti scans are somewhat aligned.
> 
> after doing that, VDB started to work perfectly. I don’t know why tbh, not sure if there is some parameter set in their code that detects the ground at a certain pre-defined height or something. Would be great if the authors could say anything about this.

OK，Thank you very much.
